src/simperium/http-request.js

```diff
@@ -0,0 +1,25 @@
+// @flow
```

Comment:
Flow is unhappy with this file (hence the CI failures), this does the trick for me:

```diff
diff --git a/src/simperium/http-request.js b/src/simperium/http-request.js
index 005d1e4..314a510 100644
--- a/src/simperium/http-request.js
+++ b/src/simperium/http-request.js
@@ -1,7 +1,7 @@
 // @flow
 import { request } from 'https'
 
-export default function( endpoint, body, options ) {
+export default function( endpoint: string, body: ( string|Buffer ), options: https$requestOptions ): Promise<string> {
     return new Promise( ( resolve, reject ) => {
         const req = request( options, res => {
             let responseData = '';

```